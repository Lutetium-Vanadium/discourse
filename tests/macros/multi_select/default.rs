fn main() {
    requestty::questions![MultiSelect { default: todo!() }];
}
