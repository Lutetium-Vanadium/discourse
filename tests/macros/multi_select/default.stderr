error: option `default` does not exist for kind `multi_select`
 --> $DIR/default.rs:2:41
  |
2 |     requestty::questions![MultiSelect { default: todo!() }];
  |                                         ^^^^^^^
